create styling for search form and search results

ooneill547 opened the pull request.

David-Corbett commented on src/scss/3_components/form/search-form.scss:
```diff
@@ -0,0 +1,130 @@
+    padding: 8px;
+  }
+
+  input:focus {
```
Remove this rule as it will have consequences for wcag.

David-Corbett commented on src/scss/3_components/form/search-form.scss:
```diff
@@ -0,0 +1,130 @@
+  }
+}
+
+%search-form {
```
Remove the silent placeholder, as it serves no purpose now that we creates singles css files. Also tends to bloat css, can amalgamate into single rules.

ooneill547 commented:
I hope I have understood your comments correctly and implemented the proper changes. I also added a preprocess hook to the .theme file based on the nidirect theme, but edited it to use the id's for news and publication search as they appear in the Cyber Security D8 site.
